### PR TITLE
Adds makeshift armor

### PR DESCRIPTION
**Pull request by BatElite**
[feat][balance]
## About the PR 
This PR adds makeshift armor, obtained by using a welder on a standard cyborg chest. It's meant to be a worse counterpart of the regular armor vests, except that they're much more easily obtained.
makeshiftpreview

## Why's this needed? 
Fills a defensive niche on the improvised combat gear thing. Maybe people could charge nukies with at least a hint of protection (provided robotics starts handing them out like candy), or loyal & rev mobs use them to get the edge in fights.

## Changelog

```
(u)BatElite:
(+)Standard cyborg chests can now be turned into makeshift armor with a welder.
```

### CODE REVIEW

**Comment by BatElite** [comment]
The stats on this thing are a stab in the dark, but I tried to fit it in between armor vests and regular space suits. Peeps with more of a feel for combat-related things are free to leave a comment.

**Review by Tarmunora — APPROVED** [review]
comperable to paramedic suit (3 melee, 0.9 ranged), but with moveslow to compensate for being able to just make them instead of having to get one from limited supply. Seems like it would be fairly balanced

**Comment by MarkNais** [comment]
Trying to place myself in the mind of a newby robotics who read the wiki...
The cyborg chest stores wires, and a battery. When on an assembled 'borg: it can be opened with the crowbar & screwdriver, or (eventually) removed with the wrench. Welding any part of the borg is done to repair brute damage...
I was initially thinking maybe a different tool would be more appropriate, but no, welder seems fine.

**Comment by BatElite** [comment]
That can get kinda convoluted yeah. I guess technically you can't mend standard chests anymore doing things this way, but I'm not sure if you ever do that anymore, Certainly I've never repaired anything that wasn't attached to a borg.

On that note I did try repairing a borg with chest damage to make sure it didn't start making armour inside the borg, but that seems to work fine.

File: code/obj/item/mob_parts/robot_parts.dm [hunk 1]
```diff
@@ -460,6 +460,16 @@
 			var/obj/item/cable_coil/cut/C = new /obj/item/cable_coil/cut(src.loc)
 			C.amount = src.wires
 			src.wires = 0
+
+		else if (isweldingtool(W) && src.type == /obj/item/parts/robot_parts/chest)
+			var/obj/item/weldingtool/welder = W
+			if (welder.try_weld(user, 3, 3))
+				var/obj/item/clothing/suit/armor/makeshift/R = new /obj/item/clothing/suit/armor/makeshift(get_turf(user))
+				boutput(user, "<span class='notice'>You remove the internal support structures of the [src.name]. It's structural integrity is ruined, but you could squeeze into it now.</span>")
```
Comment (ZeWaka) [inline code/obj/item/mob_parts/robot_parts.dm:468]:
```suggestion
				boutput(user, "<span class='notice'>You remove the internal support structures of the [src]. It's structural integrity is ruined, but you could squeeze into it now.</span>")
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**